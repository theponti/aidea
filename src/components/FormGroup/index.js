import t from 'prop-types'
import React from 'react'
import styles from './FormGroup.module.scss'

export function FormGroup ({ children }) {
  return <div className={styles.container}>{children}</div>
}

FormGroup.propTypes = {
  children: t.node
}

export default FormGroup
