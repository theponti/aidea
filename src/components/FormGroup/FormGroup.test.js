import React from "react";
import { render } from "@testing-library/react";
import FormGroup from ".";

describe("<FormGroup/>", () => {
  it("should render component", () => {
    const { container } = render(<FormGroup />);
    expect(container).toMatchSnapshot();
  });
});
