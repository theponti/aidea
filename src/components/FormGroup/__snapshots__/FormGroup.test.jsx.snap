// Vitest Snapshot v1

exports[`<FormGroup/> > should render component 1`] = `
<div>
  <div
    class="_container_er7mq_1"
  />
</div>
`;
